packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -765,6 +765,7 @@ function Panel({
         onOpen={onOpen}
         onClose={onClose}
         overlayProps={{
+          id: id ? `${id}--dialog` : undefined,
```

Comment:
Did you need to make this change to SelectPanel? Wondering if you could've passed an `id` from your story in `overlayProps` instead.

Reply:
You're right. Can do with just `overlayProps`

Reply:
Nice, thanks! Was wondering because if we wanted to go this route of loading more items on scroll, not having to modify SelectPanel here would mean we could do that today, without needing a primer/react update first.

Reply:
perfect! there is however another bug here :( 

Known bug: Scroll resets to top when the items change. Works well when `primer_react_select_panel_remove_active_descendant` feature flag is on. I've asked what's the rollout plan [on slack](https://github.slack.com/archives/C087S9D3W87/p1755848768159079)